react/css/src/digitv2/components/formCardV2.scss

```
@@ -52,77 +52,77 @@
         
           &.with-shadow,
           &.without-footer{
-            padding: theme("digitv2.spacers.spacer4");
+            padding: var(--digitv2-spacers-spacer4);
           }
         }
 
         &.with-divider{
           @media (min-width: 48rem) {
-            grid-column-gap: theme("digitv2.spacers.spacer12");
+            grid-column-gap: var(--digitv2-spacers-spacer12);
```

Stylelint (17.3.0)

[error] Expected "grid-column-gap" to be "column-gap" (property-no-deprecated)

(property-no-deprecated)